### DISCUSSION TITLE
Why was "Your Codespaces" removed from the User Menu?

### DISCUSSION THREAD

**Discussion by Skyrant** [community/community#41040] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Github in a Browser

Last time i used codespaces i had an option in the top right main menu to go to the codespaces page. I don't see any way to get to my active codespaces page other than typing https://github.com/codespaces

Why was this removed from the main menu where it belongs?

**Comment by ossanna16** [comment]
Hi there @Skyrant and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by spenserblack** [comment]
> I don't see any way to get to my active codespaces page other than typing

Is it not in the top-left of the page, between the "Issues" and "Marketplace" links?
![screen](https://user-images.githubusercontent.com/8546709/206471174-aea3b394-15b9-42ff-9e41-4b1b36cfbc54.png)

(I also didn't notice this change at first, and was manually typing the URL :laughing:)

**Comment by aletsdelarosa** [comment]
Hello!

If you can't see the menu like the one in @spenserblack 's comment. You can click on your profile icon. It appears as "Your Codespaces"
